test/e2e/case2_status_sync_test.go

```diff
@@ -117,7 +119,7 @@ var _ = Describe("Test status sync", func() {
 			return hubPlc.Object["status"]
 		}, defaultTimeoutSeconds, 1).Should(Equal(managedPlc.Object["status"]))
 	})
-	It("Should set status to Compliant again", func() {
+	It("Should set status to Compliant again", func(ctx context.Context) {
```

Comment:
(Optional) Use Ginkgo's `SpecContext`
ref: https://onsi.github.io/ginkgo/#interruptible-nodes-and-speccontext
```suggestion
	It("Should set status to Compliant again", func(ctx SpecContext) {
```